10-deploying-an-application/README.md

```diff
@@ -0,0 +1,465 @@
+# Deploying an application with a plan
+
+> **Difficulty**: Intermediate
+
+> **Time**: Approximately 10 minutes
+
+In this exercise you will further explore Puppet Plans by writing a
+multi-stage plan to deploy a dummy application.
+
+The dummy application for this lesson consists of 4 nodes. It has a
+single database node, 2+ app servers, and a single load balancer in front of
+them. When a new version of the application is released the following steps must be carried out:
+
+1. Install Application code on app and db servers. `my_app::install`
+1. Run migrations on the db server. `my_app::migrate`
+1. For each application server.
+   1. Check connections on load balancer. `my_app::lb`
+   1. Drain connections on load balancer. `my_app::lb`
+   1. Restart with new code version. `my_app::deploy`
+   1. Run a Health check. `my_app_healthcheck`
+   1. Add back to the load balancer. `my_app::lb`
+1. Clean up the old version of the application
+
+
+# Prerequisites
+
+For the following exercises you should already have `bolt` installed and have 4
+linux nodes available to run commands against. The following guides will help:
+
+1. [Acquiring Nodes](../2-acquiring-nodes)
+1. [Writing tasks](../5-writing-tasks)
+1. [Writing Advanced Plans](../9-writing-advanced-plans)
+
+### Acquire nodes and set up inventory
+
+This lesson requires 4 nodes. If you set up nodes with docker or vagrant in [lesson 2](../2-acquiring-nodes). You'll need to provision an extra node.
```

Comment:
Not sure what "You'll need to provision an extra node." means in the context? Do I need 4 nodes or 5?